cssom-1/Overview.bs

```diff
@@ -2034,6 +2034,25 @@ Note: The serialization of an empty CSS declaration block is the empty string.
 Note: The serialization of a non-empty CSS declaration block does not include any surrounding whitespace, i.e., no whitespace appears
 before the first property name and no whitespace appears after the final semicolon delimiter that follows the last property value.
 
+A <a>CSS declaration block</a> has these <a>attribute change steps</a> for its <a for="CSSStyleDeclaration">owner node</a>:
+<ol>
+ <li>If the <a for="CSSStyleDeclaration">readonly flag</a> is set, terminate these steps.
```

Comment:
How can this even happen?  The cases when there is an owner node are currently not readonly and there are no plans I am aware of to make them readonly...

There should probably be a note here explaining what this test is about, because it's pretty confusing.

Reply:
getComputedStyle sets owner node per the spec. But maybe that is a spec bug?

https://drafts.csswg.org/cssom/#dom-window-getcomputedstyle

Reply:
Oh, I missed that somehow.  OK, then this is probably fine.  That said, this is the first actual _use_ of owner node that I can see.  So if it simplifies things, we could probably avoid setting owner node in `getComputedStyle` instead of dealing with it here....

Reply:
Yeah I think we can set it to null there.

Reply:
We can discuss this in a separate PR. I don't think that needs to block this change.

Actually, I have a feeling that having the declaration block from `getComputedStyle` have the owner node set makes sense. Not sure how useful it is, though.